README.md

```diff
@@ -170,20 +180,87 @@ platform, but it's also been tested on the OrangeCrab and ButterStick.
 
    For the Arty A7 A35, set `FLASH_ADDRESS` to `0x300000` and pass `-f a35`.
    ```
-   microwatt/openocd/flash-arty -f a100 build/microwatt_0/arty_a7-100-vivado/microwatt_0.bit
-   microwatt/openocd/flash-arty -f a100 dtbImage.microwatt.elf -t bin -a $FLASH_ADDRESS
+   sudo microwatt/openocd/flash-arty -f a100 build/microwatt_0/arty_a7-100-vivado/microwatt_0.bit
+   sudo microwatt/openocd/flash-arty -f a100 dtbImage.microwatt.elf -t bin -a $FLASH_ADDRESS
    ```
 
 5. Connect to the second USB TTY device exposed by the FPGA
 
    ```
-   minicom -D /dev/ttyUSB1
+   sudo minicom -D /dev/ttyUSB1
    ```
+   If the cable was plugged in, unplug it when you're done and you'll see the serial console on the minicomputer. If you can't see the serial console, see below.
```

Comment:
This statement confuses me. If you unplug the cable, how are you going to see the serial console?